facia-tool/app/controllers/FaciaContentApiProxy.scala

```diff
@@ -35,10 +35,29 @@ object FaciaContentApiProxy extends Controller with Logging with ExecutionContex
     }
   }
 
+  def capiLive(path: String) = APIAuthAction.async { request =>
```

Comment:
Can we call the above one `capiDraft`? And maybe the endpoint too just to make it clear.